Makefile

```
@@ -0,0 +1,18 @@
+export DOCKER_BUILDKIT=1
+
+DOCKER_BUILD_ARGS ?=
+
+ifneq ($(PHP_VERSION),)
+DOCKER_BUILD_ARGS += --build-arg PHP_VERSION=$(PHP_VERSION)
+endif
+
+DOCKER_BUILD_IID_FILE = .docker-$(PHP_VERSION).iid
```

```suggestion
DOCKER_BUILD_IID_FILE = .docker-php$(PHP_VERSION).iid
```
Otherwise when the version isn't explicitly passed you end up with a weirdly named filename.